### PR TITLE
Add Accedian MetroNID OS Support

### PR DESCRIPTION
**Pull request by thecityofguanyu**
Committer: <thecityofguanyu@outlook.com>

 On branch thecityofguanyu/add-accedian-os
 Changes to be committed:
	new file:   html/images/os/accedian.png
	modified:   includes/definitions.inc.php
	new file:   includes/discovery/os/metronid.inc.php
	new file:   includes/discovery/processors/metronid.inc.php
	new file:   includes/discovery/sensors/temperatures/metronid.inc.php
	new file:   includes/polling/os/accedian.inc.php
	new file:   mibs/ACCEDIAN-SMI.mib
	new file:   mibs/ACD-ALARM-MIB.mib
	new file:   mibs/ACD-CFM-MIB.mib
	new file:   mibs/ACD-DESC-MIB.mib
	new file:   mibs/ACD-DISCOVERY-MIB.mib
	new file:   mibs/ACD-FILTER-MIB.mib
	new file:   mibs/ACD-PAA-MIB.mib
	new file:   mibs/ACD-POLICY-MIB.mib
	new file:   mibs/ACD-PORT-MIB.mib
	new file:   mibs/ACD-REGULATOR-MIB.mib
	new file:   mibs/ACD-SA-MIB.mib
	new file:   mibs/ACD-SFP-MIB.mib
	new file:   mibs/ACD-SHAPER-MIB.mib
	new file:   mibs/ACD-SMAP-MIB.mib
	new file:   mibs/ACD-TID-MIB.mib

### CODE REVIEW

File: includes/discovery/os/metronid.inc.php [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+
+
+if (!$os) {
+    if (preg_match('/AMN-/', $sysDescr)) {
+        $os = 'accedian';
```
Comment (laf) [inline includes/discovery/os/metronid.inc.php]: Is this the right name for the OS? Accedian is the company name and detection should be named after what the OS is.

File: includes/discovery/processors/metronid.inc.php [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+
+
+
+if ($device['os'] == 'accedian') {
+ if ($device['version'] == 'AEN_5.3.1_22558') {
```
Comment (laf) [inline includes/discovery/processors/metronid.inc.php]: Indents are four spaces.

**Comment by laf** [comment]
Slight formatting issue in includes/discovery/processors/metronid.inc.php

Also query about the $os name.

**Comment by thecityofguanyu** [comment]
@laf 

>Slight formatting issue in includes/discovery/processors/metronid.inc.php

Fixed in latest commit.

>Also query about the $os name.

Accedian made it a bit unclear here. The closest I can find to an OS name is 'AEN', which shows under firmware version. I've update it to reflect that instead of Accedian, also fixed in latest commit.

**Comment by laf** [comment]
Didn't spot this the first time round but the files metronid.inc.php need updating to be the $os.inc.php

**Comment by thecityofguanyu** [comment]
@laf 

>Didn't spot this the first time round but the files metronid.inc.php need updating to be the $os.inc.php

Fixed.

**Pull request merged**